pkg/app/server/service/webservice/service.proto

```diff
@@ -96,6 +96,9 @@ service WebService {
 
     // Events
     rpc ListEvents(ListEventsRequest) returns (ListEventsResponse) {}
+
+    // ControlPlain metadata
+    rpc GetControlPlainMeta(GetControlPlainMetaRequest) returns (GetControlPlainMetaResponse) {}
```

Comment:
I guess without implementation for this rpc, the `build` will not pass. Should add implementation for it in `web_api.go` too.

Reply:
Thank you. I'll add it